src/main/java/racinggame/RacingMain.java

```diff
@@ -0,0 +1,34 @@
+package racinggame;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import racinggame.SnapShotStore.SnapShot;
+
+public class RacingMain {
+
+    public static void main(String[] args) {
+        InputDto read = InputView.readCount();
+        RacingGame racingGame = new RacingGame(read.getCarCount());
+        racingGame.start(radoms(read.getRepeatCount(), read.getCarCount()));
```

Comment:
```suggestion
        RacingGame racingGame = new RacingGame(readDto);
        SnapShot result = racingGame.start();
```
레이싱을 진행하는 데 필요한 값들은 생성자를 통해 초기화하는 것이 더 좋을 것 같아요
`RacingGame` 객체를 생성할 때 필요한 상태를 명확히 설정할 수 있습니다.
이후 경기를 진행하고 곧 바로 결과를 받아볼 수 있게도 바꿔보면 좋을 것 같아요.


세터와 게터를 통해 내부 상태를 직접 조작하고 가져오는 방식보다는 
메서드를 통해 외부에서 객체의 내부 구현을 알 필요 없이 사용할 수 있게 만들어보면 좋을 것 같습니다.

Reply:
개인적으로readDto를 인자로 넘기면 너무 readDto에 의존하게 되는 것 같습니다. 하지만 확실히 생성자로 넘기는편이 더 좋아보이기는 합니다.

Reply:
DTO에 의존하면 안 되는 이유가 있을까요? DTO라는 이름 때문일까요?

RacingGame 설정에 필요한 값이라고 생각한다면 저는 의존해도 된다고 생각해요.
지훈님의 의견도 들어보고 싶습니다! 😄

Reply:
개인적인 생각이긴한데요.InputDto의 역할이 InputView에서 생성된 데이터에 관한 데이터를 이동시키는 역할인데 여기서 만약 InputDto를 의존하게 된다면 InputView의 변화로 인한 영향이 RadomGame까지 넘어가는게 아닌가 싶은 생각이 듭니다! 그래서 만약의존 할꺼라면 Input과 관련된 헬퍼 객체를 따로구현하는 쪽이 괜찮지 않을까 싶은데 어떻게 생각하시나요?

Reply:
이름이 InputDto라 그런걸까요?
그럼 이름을 바꿔 `GameStartRequest` 같은 이름으로 변경 후 해당 객체의 위치를 바꿔본다면 어떨까요?